src/Products/ProductHydrator.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace FurnitureStoreApi\Products;
+use \PDO;
+
+class ProductHydrator
+{
+    public static function getProductsByCategory($PDO, $db)
```

Comment:
Looks like scope creep again: is this needed yet?

Also, looks like you have two parameters here: `$PDO` and `$db`. Is this what you want? And how are they different?

Reply:
Yes, we realised as we did peer review but will leave it in until we get to that Story.  If we can do that?